src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php

```diff
@@ -101,7 +101,7 @@ public function count()
         } elseif ($numResults == 1) {
             $result = array_values($result[0]);
 
-            return $result[0];
+            return (int) $result[0];
```

Comment:
What would that advantage of `intval()` be? I would also use type casting by default.